diagrams/request_uri_mode_post_through_browser_api.md

```diff
@@ -0,0 +1,57 @@
+```plantuml
```

Comment:
what happens to these files? asking for a friend

Reply:
same question :)

Reply:
I checked this with Torsten - he would like these diagrams to be in the specification, but wasn't sure how to actually achieve that.

I've opened an issue - hopefully someone might be able to help us achieve that without any manual steps: https://github.com/openid/OpenID4VP/issues/206